database/index.js

```diff
@@ -0,0 +1,72 @@
+/*
+- create a folder then change path in the terminal/cmd to that folder and execute the command 'npm install mysql'
+- create an 'index.js' file in the same folder and copy the following code in it.
+- zip the folder and upload it to AWS Lambda.
+*/
+// Following code is based on the assumption that the table structure consists of a column named 'last_modified(timestamp)' on the basis of which incremental update will happen
+// and 'isDeleted(int11)' whose value defines if that particular row has been deleted or not (0 - not deleted, 1 - deleted).
+var mysql = require('mysql');
+exports.handler =  (request, context, callback) => {
```

Comment:
plz add empty lines between blocks so it's easier to read. Let's add some air to the code :)